packages/server/src/schema/uuid/abstract-entity/resolvers.ts

```diff
@@ -48,6 +56,64 @@ export const resolvers: InterfaceResolvers<'AbstractEntity'> &
       return entityRevision.__typename
     },
   },
+  EntityQuery: {
+    async deletedEntities(_parent, payload, { dataSources }) {
+      const LIMIT = 1000
+      const { first = 100, after, instance } = payload
+
+      if (first > LIMIT)
+        throw new UserInputError(`'first' may not be higher than ${LIMIT}`)
+
+      const deletedAfter = after
+        ? (
+            JSON.parse(Buffer.from(after, 'base64').toString()) as {
+              dateOfDeletion: string
+            }
+          ).dateOfDeletion
+        : undefined
+
+      if (deletedAfter && !isDateString(deletedAfter))
+        throw new UserInputError(
+          'the encoded dateOfDeletion  in `after` should be a date'
+        )
+
+      const { deletedEntities } =
+        await dataSources.model.serlo.getDeletedEntities({
+          first,
```

Comment:
```suggestion
          first + 1,
```

Otherwise `hastNexrPage` would be always `false`